packages/action-bar/stories/action-bar-scoped.stories.ts

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { html, TemplateResult } from '@spectrum-web-components/base';
+
+import '@spectrum-web-components/action-bar/sp-action-bar-scoped.js';
+import '@spectrum-web-components/field-label/sp-field-label.js';
+import '@spectrum-web-components/action-button/sp-action-button.js';
+import '@spectrum-web-components/action-group/sp-action-group.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-edit.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-more.js';
+
+export default {
+    component: 'sp-action-bar',
+    title: 'Action Bar/Scoped',
+};
+
+export const Default = (): TemplateResult => {
```

Comment:
Having these stories be hard duplicates will be difficult to maintain. Do you have any thoughts on wrappers or approaches that would allow us to use a single set of stories and have then surfaced in both scoped and un-scoped versions?

Reply:
I think there are two ways we can do this. Either using global dropdown or using props at individual level. I have implemented the later for now as limited only to action-bar but it could be moved globally with little changes.